cumulus_library/studies/discovery/code_detection.py

```diff
@@ -0,0 +1,84 @@
+""" Module for generating encounter codeableConcept table"""
+
+from cumulus_library.base_table_builder import BaseTableBuilder
+from cumulus_library.helper import get_progress_bar, query_console_output
+from cumulus_library.template_sql.templates import get_code_system_pairs
+from cumulus_library.template_sql.utils import (
+    is_codeable_concept_array_populated,
+    is_codeable_concept_populated,
+    is_code_populated,
+)
+
+from cumulus_library.studies.discovery.code_definitions import code_list
+
+
+class CodeDetectionBuilder(BaseTableBuilder):
+    display_text = "Selecting unique code systems..."
```

Comment:
unicode nerd nit: don't you know there's a character for that?  :smile:  U+2026: … or in monospace `…`